CMakeLists.txt

```diff
@@ -182,14 +182,11 @@ endif()
 
```

Comment:
Not a change request, but for my own knowledge. Do you know if the recommendation nowadays is to handle include dirs through targets? E.g. should we have a `SpectreSrcInclude` target that we use? Similar for tests? Or should the `spectre_add_library`-type calls set the include dir for each target instead of setting global ones?

Reply:
I looked at that, but haven't figured out a great solution. We want to include headers with the full relative path to `src/`, which means the include dir is the same for all libraries. I haven't found a best-practice example for using `target_include_directories` effectively for this, so I kept the global `include_directories` for now.

Reply:
Ah right... Alright, sounds good to me :) thanks!